README.md

```diff
@@ -1,4 +1,5 @@
 # Core Vanilla JavaScript
+Grey Toucan Please see solo study Guide for my work.
```

Comment:
Hey there,
There's nothing here that I can review. You might want to go back through the review steps and try again. 

https://playbook.learnersguild.org/Learning_Guide/Review.html

there's a step by step guild on how to do that. I apologize if I'm just not finding the changes. If they're there, let me know and I'll review again.